dts: update memory map and remove ext-uicr

hakonfam opened the pull request:
Extended UICR will not be used as its configurations will be merged
with the UICR registers in NVR.

Memory maps changes are needed to align with pre compiled
firmware.

hakonfam committed:
dts: update memory map and remove ext-uicr

Extended UICR will not be used as its configurations will be merged
with the UICR registers in NVR.

Memory maps changes are needed to align with pre compiled
firmware.

Signed-off-by: Håkon Amundsen <haakon.amundsen@nordicsemi.no>
Signed-off-by: Jonathan Nilsen <jonathan.nilsen@nordicsemi.no>

jonathannilsen commented on boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp_0_8_0.overlay:
```diff
@@ -0,0 +1,7 @@
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nrf54h20dk_nrf54h20_0_8_0.dtsi>
```
Does this work for you? For me, this resulted in a build error, whereas
```suggestion
#include "nrf54h20dk_nrf54h20_0_8_0.dtsi"
```
works

anangl commented on dts/common/nordic/nrf9280.dtsi:
```diff
@@ -113,14 +113,12 @@
 			compatible = "nordic,nrf-uicr-v2";
 			reg = <0xfffa000 DT_SIZE_K(2)>;
 			domain = <3>;
-			ptr-ext-uicr = <&cpurad_uicr_ext>;
```
Why aren't these nodes removed as in `nrf54h20.dtsi`?

hakonfam replied:
removed them now

carlescufi committed:
ci: Switch to CI image v0.27.4

This commit updates the CI workflows to use the CI image v0.27.4, which
includes nrf-regtool v7.0.0.

Signed-off-by: Carles Cufi <carles.cufi@nordicsemi.no>
Signed-off-by: Stephanos Ioannidis <root@stephanos.io>

carlescufi commented:
Tested here: https://github.com/zephyrproject-rtos/zephyr-testing/pull/286
